target/linux/x86/64/config-6.6

```diff
@@ -211,6 +211,9 @@ CONFIG_FONT_SUPPORT=y
 CONFIG_FRAMEBUFFER_CONSOLE=y
 CONFIG_FRAMEBUFFER_CONSOLE_DETECT_PRIMARY=y
 # CONFIG_FRAMEBUFFER_CONSOLE_ROTATION is not set
+# CONFIG_DRM_ACCEL_HABANALABS is not set
+# CONFIG_DRM_ACCEL_IVPU is not set
+# CONFIG_DRM_ACCEL_QAIC is not set
 CONFIG_FREEZER=y
 CONFIG_FUSION_SAS=y
 CONFIG_FW_CACHE=y
```

Comment:
Please order this file correctly, you can use `./scripts/kconfig-reorder.sh`.

Reply:
OK, updated